src/Shelly.hs

```diff
@@ -850,6 +850,11 @@ print_stderr shouldPrint a =
 print_commands :: Bool -> Sh a -> Sh a
 print_commands shouldPrint a = sub $ modify (\st -> st { sPrintCommands = shouldPrint }) >> a
 
+
+-- | Create a sub-Sh in which commands are sent to the user-defined function.
```

Comment:
This needs a `@since 1.12.1` annotation.  Append
```haskell
--
-- @since 1.12.1
```
(haddock formatting restrictions).

Reply:
Appended since